components/molecule/imageEditor/src/index.js

```diff
@@ -99,6 +113,7 @@ const MoleculeImageEditor = ({
 MoleculeImageEditor.displayName = 'MoleculeImageEditor'
 MoleculeImageEditor.propTypes = {
   aspect: PropTypes.number,
+  previewOnlyMode: PropTypes.bool,
```

Comment:
```suggestion
  isPreviewMode: PropTypes.bool,
```

or even

```suggestion
  isPreview: PropTypes.bool,
```

given they're boolean props. What do you think...?